tfe/provider.go

```diff
@@ -68,6 +68,7 @@ func Provider() terraform.ResourceProvider {
 			"tfe_team_access":             dataSourceTFETeamAccess(),
 			"tfe_workspace":               dataSourceTFEWorkspace(),
 			"tfe_workspace_ids":           dataSourceTFEWorkspaceIDs(),
+			"tfe_oauth_client":            dataSourceTFEOAuthClient(),
```

Comment:
nit: this should be moved up above organization membership so things are in alphabetical order